soccer/gameplay/tactics/positions/wing_defender.py

```diff
@@ -0,0 +1,76 @@
+import single_robot_behavior
```

Comment:
Is this part of the AdaptiveDefense code that should be pulled out of this PR @JNeiger

Reply:
Yes